core/css.js

```diff
@@ -440,10 +450,18 @@ Blockly.Css.CONTENT = [
 
   '.blocklyTreeSeparator {',
     'border-bottom: solid #e5e5e5 1px;',
-    'height: 0px;',
+    'height: 0;',
     'margin: 5px 0;',
   '}',
 
+  '.blocklyTreeSeparatorHorizontal {',
+    'border-right: solid #e5e5e5 1px;',
+    'width: 0;',
+    'padding: 5px 0;',
```

Comment:
just curious - why the mix of padding and margin and not just either padding all around or margin all around?